package.json

```diff
@@ -47,7 +47,8 @@
     "standard-version": "^4.2.0",
     "style-loader": "^0.18.2",
     "webpack-dev-server": "^2.4.1",
-    "webpack-merge": "^4.0.0"
+    "webpack-merge": "^4.0.0",
+    "webpack": "^3.11.0"
```

Comment:
Why is there a dev dependency on `webpack`?

Reply:
@jhnns need for local tests, remove from PR?

Reply:
@evilebottnawi Can you remove it for now please ? :)

Reply:
@michael-ciniawsky let's do it :+1: